Assets/Game/Scripts/Character/AnimationTrigger.cs

```diff
@@ -1,8 +1,8 @@
-﻿public enum AnimationTrigger
+﻿using UnityEngine;
+
+public class AnimationTrigger
 {
-  Move,
-  Idle,
-  Attack,
-  Die,
-  Hurt
+  public static readonly int dead = Animator.StringToHash("Dead");
+  public static readonly int attack = Animator.StringToHash("Attack");
+  public static readonly int run = Animator.StringToHash("Run");
```

Comment:
Should we have "hit" or "hurt" as well?

Reply:
Yes, there should be idle too. But I leave it out for whoever (most likely Aaron) going to work on controlling those two animations to add in it.